packages/components/src/components/table/table-head-cell/table-head-cell-styles.ts

```diff
@@ -28,39 +25,28 @@ export const getComponentCss = (
     '@global': {
       ':host': addImportantToEachRule({
         display: 'table-cell',
-        padding: `${pxToRemWithUnit(2)} ${pxToRemWithUnit(12)} ${pxToRemWithUnit(8)}`,
-        borderBottom: `1px solid ${contrastMediumColor}`,
+        padding: `2px ${spacingFluidSmall} ${spacingFluidSmall} ${spacingFluidSmall}`,
+        borderBottom: `1px solid var(${cssVariableTableBorderColor}, ${getThemedColors('light').contrastLowColor})`,
         verticalAlign: 'bottom',
         fontWeight: fontWeightSemiBold,
         whiteSpace: multiline ? 'normal' : 'nowrap',
         ...hostHiddenStyles,
       }),
       ...(sortable
         ? {
-            button: {
-              display: 'flex',
-              alignItems: 'flex-end',
-              padding: 0,
-              boxSizing: 'border-box',
-              appearance: 'none',
-              border: 'none',
-              ...textSmallStyle,
-              fontWeight: fontWeightSemiBold,
-              color: primaryColor,
-              textDecoration: 'none',
-              textAlign: 'left',
-              background: 'transparent',
-              cursor: 'pointer',
-              ...getFocusJssStyle({ offset: 1 }),
+            '.button-pure': {
               ...hoverMediaQuery({
-                ...getHoverJssStyle(),
                 '&:hover, &:focus': {
                   '& .icon': {
                     opacity: 1,
```

Comment:
transition definition is missing for opacity

Reply:
Fixed it